observe/observe.go

```diff
@@ -0,0 +1,110 @@
+// Package observe provides functions
+// that help with setting tracing/metrics
+// in cloud providers, mainly GCP.
+package observe
+
+import (
+	"context"
+	"os"
+
+	traceapi "cloud.google.com/go/trace/apiv2"
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"contrib.go.opencensus.io/exporter/stackdriver/monitoredresource"
+	"go.opencensus.io/exporter/prometheus"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+// NewStackDriverExporter will return the tracing and metrics through
+// the stack driver exporter, if exists in the underlying platform.
+// If exporter is registered, it returns the exporter so you can register
+// it and ensure to call Flush on termination.
+func NewStackDriverExporter(projectID string, onErr func(error)) (*stackdriver.Exporter, error) {
+	_, svcName, svcVersion := GetServiceInfo()
+	opts := getSDOpts(projectID, svcName, svcVersion, onErr)
+	if opts == nil {
+		return nil, nil
+	}
+	return stackdriver.NewExporter(*opts)
+}
+
+// NewPrometheusExporter return a prometheus Exporter for OpenCensus.
+func NewPrometheusExporter(opts prometheus.Options) (*prometheus.Exporter, error) {
```

Comment:
Ok last thing, sorry...I imagine we dont need this one at all, right? It's not giving us any value at this point.

Reply:
@jprobinson yesss that was the point i was trying to make in calling this `gcputil` 😄 but im totally happy with just `observe` for now

Reply:
@jprobinson will delete this